model/jpa/src/main/resources/META-INF/jpa-changelog-3.2.0.xml

```diff
@@ -0,0 +1,235 @@
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_COMPONENT_REALM" tableName="COMPONENT">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_COMPO_CONFIG_COMPO" tableName="COMPONENT_CONFIG">
+            <column name="COMPONENT_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_COMPOSITE" tableName="COMPOSITE_ROLE">
+            <column name="COMPOSITE" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_COMPOSITE_CHILD" tableName="COMPOSITE_ROLE">
+            <column name="CHILD_ROLE" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CREDENTIAL_ATTR_CRED" tableName="CREDENTIAL_ATTRIBUTE">
+            <column name="CREDENTIAL_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_FED_CRED_ATTR_CRED" tableName="FED_CREDENTIAL_ATTRIBUTE">
+            <column name="CREDENTIAL_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_GROUP_ATTR_GROUP" tableName="GROUP_ATTRIBUTE">
+            <column name="GROUP_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_GROUP_ROLE_MAPP_GROUP" tableName="GROUP_ROLE_MAPPING">
+            <column name="GROUP_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_GROUP_ROLE_MAPP_ROLE" tableName="GROUP_ROLE_MAPPING">
+            <column name="ROLE_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_IDENT_PROV_REALM" tableName="IDENTITY_PROVIDER">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_ID_PROV_CONF_ID_PROV" tableName="IDENTITY_PROVIDER_CONFIG">
+            <column name="IDENTITY_PROVIDER_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_ID_PROV_MAPP_REALM" tableName="IDENTITY_PROVIDER_MAPPER">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_IDP_MAPP_CONFIG_IDP_MAPP" tableName="IDP_MAPPER_CONFIG">
+            <column name="IDP_MAPPER_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_KEYCLOAK_GROUP_REALM" tableName="KEYCLOAK_GROUP">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_KEYCLOAK_ROLE_CLIENT" tableName="KEYCLOAK_ROLE">
+            <column name="CLIENT" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_KEYCLOAK_ROLE_REALM" tableName="KEYCLOAK_ROLE">
+            <column name="REALM" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_POLICY_CONF_POLICY" tableName="POLICY_CONFIG">
+            <column name="POLICY_ID" type="VARCHAR(36)"/>
```

Comment:
This index is unnecessary because there is already an index for primary key on POLICY_CONFIG(**POLICY_ID**, NAME) named _constraint_dpc_. Please remove.